src/Components/Products/products.css

```diff
@@ -34,10 +34,36 @@
     margin-right: 10vw;
 }
 
+.pencil2 {
+    margin-right: 2vw;
+}
+
+.trash {
+    margin-right: 2vw;  
+}
+
 .UpButton {
     padding: 5px;
 }
 
 .AmountPlace {
     width: 50px;
-}
\ No newline at end of file
+}
+
+.ListInput2{
```

Comment:
Da sich ListInput2 auf die Anzahl eines Eintrags bezieht, wäre vielleicht ein Name wie count besser geeignet. außerdem möchte ich sie ermutigen lieber die BEM-Schreibweise für CSS-Klassen zu verwenden, um etwaige namenskollisionen und Performance-probleme zu vermeiden:
http://getbem.com/introduction/

in diesem fall wäre daher der Klassenname eher
Product--count